src/tartiflette_asgi/_endpoints.py

```diff
@@ -33,16 +33,36 @@ async def get(self, request: Request) -> Response:
 
 class GraphQLEndpoint(HTTPEndpoint):
     async def get(self, request: Request) -> Response:
-        return await self._get_response(request, data=request.query_params)
+        variables = None
+        try:
+            if "variables" in request.query_params:
+                variables = json.loads(request.query_params["variables"])
+        except json.JSONDecodeError:
+            return JSONResponse(
+                {"error": "Unable to decode variables: Invalid JSON."}, 400
+            )
```

Comment:
(Nit) Swap if and try/except blocks so try/except is as closed as possible to potential exception source:

```suggestion
        if "variables" in request.query_params:
            try:
                variables = json.loads(request.query_params["variables"])
            except json.JSONDecodeError:
                return JSONResponse(
                    {"error": "Unable to decode variables: Invalid JSON."}, 400
                )
```